model/role.go

```diff
@@ -152,7 +159,7 @@ func init() {
 	}
 
 	SystemManagerDefaultPermissions = []string{
-		PERMISSION_SYSCONSOLE_READ_ABOUT.Id,
+		PERMISSION_SYSCONSOLE_READ_ABOUT_EDITION_AND_LICENSE.Id,
```

Comment:
Is the system manager not supposed to have PERMISSION_SYSCONSOLE_WRITE_ABOUT_EDITION_AND_LICENSE?

Reply:
According to this doc, https://mattermost.atlassian.net/wiki/spaces/EN/pages/463798295/New+Admin+Roles+Special+Admins
System Manager is RO on `Edition and License`